contracts/pool/constant-product/ConstantProductPool.sol

```diff
@@ -40,13 +40,13 @@ contract ConstantProductPool is IPool, ERC20, ReentrancyGuard {
     uint256 public immutable swapFee;
     uint256 internal immutable MAX_FEE_MINUS_SWAP_FEE;
 
-    address public immutable barFeeTo;
     IBentoBoxMinimal public immutable bento;
     IMasterDeployer public immutable masterDeployer;
     address public immutable token0;
     address public immutable token1;
 
     uint256 public barFee;
+    address public barFeeTo;
```

Comment:
packing this var among other addresses allows us to use 1 slot in storage for all addresses